llm_on_ray/inference/models/CodeLlama-7b-hf.yaml

```diff
@@ -13,9 +13,4 @@ ipex:
 model_description:
   model_id_or_path: codellama/CodeLlama-7b-hf
   tokenizer_name_or_path: codellama/CodeLlama-7b-hf
-  chat_processor: ChatModelGptJ
-  prompt:
-    intro: ''
-    human_id: ''
-    bot_id: ''
-    stop_words: []
+  chat_template: "llm_on_ray/common/templates/template_codellama.jinja"
```

Comment:
We may need to use a relative path the current yaml file, otherwise it's hard for user to specify the file since they may put the template files elsewhere.